MongoDB/containers/0.1.0/mongodb-config/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Installing dnsutils and jq
+echo "deb http://http.debian.net/debian wheezy-backports main" | tee /etc/apt/sources.list.d/wheezy-backports.list > /dev/null
+echo "deb http://security.debian.org/ wheezy/updates main contrib non-free " | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+echo "deb-src http://security.debian.org/ wheezy/updates main contrib non-free" | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+apt-get -q update > /dev/null
+apt-get install -qqy dnsutils jq > /dev/null 2>&1
+
+# Check for lowest ID
+/opt/rancher/bin/lowest_idx.sh
```

Comment:
The problem is that dig has to run on the MongoDB container itself multiple times to get the list of ips of the containers in different situations (initialization, scaling), for example here I wait until all containers are up and then start the initialization:
```
while [ `$DIG A $MONGO_SERVICE_NAME +short | wc -l` -lt 3 ]; do
	echo 'mongo instances are less than 3.. waiting!'
	sleep 5
done
```
I tried to install dig on alpine and then link dig command to /opt/rancher/bin but it didn't work, i don't know what is the alternative for this situation.

I think i can remove jq command altogether and try to extract the metadata manually in the lowest_idx script.